### ISSUE TITLE
GoogleMapsUtils module build incompatible with latest swift version

### ISSUE THREAD

**Issue by VincentDurandChewbi** [googlemaps/google-maps-ios-utils#393]
#### Environment details

1. Mac os Big Sur Version 11.4
2. XCode version Version 13.0 (13A233)
3. GoogleMapsUtils version 4.1.0

#### Steps to reproduce

1. Add GoogleMapsUtils in my SPM Project (next major version)
2. Build app
3. Error hapend (cf screenCap below)

![MicrosoftTeams-image (43)](https://user-images.githubusercontent.com/8911520/148410793-f03b831d-5331-4cab-a001-c49b20fa4fc8.png)

**Comment by arriolac** [comment]
The library is built with library evolution support (`BUILD_LIBRARY_FOR_DISTRIBUTION` is set to `YES`) so it _should_ work on the latest version of Xcode/Swift. I did test this on Xcode 13 and was able to get it to work. Perhaps a clean build might do the trick?

**Comment by dmi3j** [comment]
Same problem with Version 13.2.1 (13C100)

It might work for the who updated Xcode from App Store for some time (I assume some old compilers might exist on local machine). But if you download latest Xcode build from App Store or Apple Developer website, the only option you have is latest compiler. **GoogleMapsUtils version 4.1.0 is not compiled on Xcode 13.2.1 (13C100)**

**Comment by edgarclerigo** [comment]
Having the same issue with Xcode 13.2.1

**Comment by arriolac** [comment]
Will take a look

**Comment by lluisgerard** [comment]
I'm having the same issue using SPM on:
- Xcode Version 15.0.1 (15A507) 
- macOS Sonoma 14.0 (23A344)
- GoogleMapUtils 4.2.2

```
Failed to build module 'GoogleMapsUtils'; this SDK is not supported by the compiler 
(the SDK is built with 'Apple Swift version 5.5.2 (swiftlang-1300.0.47.5 clang-1300.0.29.30)', 
while this compiler is 'Apple Swift version 5.9 (swiftlang-5.9.0.128.108 clang-1500.0.40.1)'). 
Please select a toolchain which matches the SDK.
```

I cleaned the project and packages, deleted deriveddata, started again... I think this should work, right? I see that `BUILD_LIBRARY_FOR_DISTRIBUTION` is true in the project. What am I missing?

**Comment by wangela** [comment]
This should be fixed with v5.0.0+ as of #444 which uses Swift 5.9.
